src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,15 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoNumbers {
+    private final List<Integer> value;
```

Comment:
로또 번호를 Integer로 관리해도 좋지만 LottoNumber라는 Value Object로 관리한다면 어떤 장점이 있을까요~?
```suggestion
    private final List<LottoNumber> value;
```